src/components/textarea/textarea.spec.tsx

```diff
@@ -23,18 +23,25 @@ import StyledTextarea from "./textarea.style";
 
 jest.mock("../../__internal__/utils/helpers/guid");
 const mockedGuid = "guid-12345";
-guid.mockImplementation(() => mockedGuid);
+(guid as jest.MockedFunction<typeof guid>).mockImplementation(() => mockedGuid);
+
+function renderTextarea(props?: TextareaProps, renderer = mount) {
+  return renderer(<Textarea name="textarea" {...props} />);
+}
 
 describe("Textarea", () => {
-  let wrapper;
+  let wrapper: ReactWrapper;
 
   testStyledSystemMargin((props) => <Textarea {...props} />);
   describe("when textarea is rendered with default props", () => {
-    let textarea;
+    let textarea: HTMLInputElement;
 
     beforeAll(() => {
       wrapper = renderTextarea();
-      textarea = wrapper.find("textarea").instance();
+      textarea = wrapper
+        .find("textarea")
+        // .instance() as unknown) as HTMLInputElement;
```

Comment:
is there a reason for commenting this line out rather than simply removing it? Git still has the history in the (unlikely) event we want to go back to using `.instance`